_overviews/hello-scala/classes-aux-constructors.md

```diff
@@ -57,7 +57,21 @@ val p4 = new Pizza
 
 We encourage you to paste that class and those examples into the Scala REPL to see how they work.
 
->Note: The `DEFAULT_CRUST_SIZE` and `DEFAULT_CRUST_TYPE` variables aren’t a great example of how to handle this situation, but because we haven’t shown how to handle enumerations yet, we use this approach to keep things simple.
+
+## Notes
+
+There are two important notes to make about this example:
+
+- The `DEFAULT_CRUST_SIZE` and `DEFAULT_CRUST_TYPE` variables are not a preferred way to handle this situation, but because we haven’t shown how to handle enumerations yet, we use this approach to keep things simple.
+- Auxiliary class constructors are a great feature, but because you can use default values for constructor parameters, you won’t need to use this feature very often. The next lesson demonstrates how using default parameter values like this often makes auxiliary constructors unnecessary:
+
+```scala
+class Pizza(
+    var crustSize: Int = DEFAULT_CRUST_SIZE, 
+    var crustType: String = DEFAULT_CRUST_TYPE
```

Comment:
I'd suggest using `ScalaStyleConstantNames` and not `JAVA_STYLE_ONES`

Reply:
Thanks, this is done!